Makefile

```diff
@@ -130,23 +130,32 @@ push-examples: $(PUSH_EXAMPLES)
 
 define test-example-template
 ifneq ("$(EXAMPLE_TESTS)", "")
-EXAMPLE_TESTS += test-example-$(1)-$(2)
+EXAMPLE_TESTS += test-example-$(1)-$(2)-$(subst .,-,$(3))
 else
-EXAMPLE_TESTS := test-example-$(1)-$(2)
+EXAMPLE_TESTS := test-example-$(1)-$(2)-$(subst .,-,$(3))
 endif
 
 .PHONY: test-example-$1-$2
-test-example-$(1)-$(2):
-	./examples/check.sh ./$1/$2 mfcp-example-$1-$2 $3 $4 $5 $6
+test-example-$(1)-$(2)-$(subst .,-,$(3)):
+	./examples/check.sh ./$1/$2 $3 mfcp-example-$1-$2 $4 $5 $6 $7
 endef
 
-$(eval $(call test-example-template,proxy,mountpoint-s3,starter,/test.txt,busybox,/data/test.txt))
-$(eval $(call test-example-template,proxy,goofys,starter,/test.txt,busybox,/data/test.txt))
-$(eval $(call test-example-template,proxy,s3fs,starter,/test.txt,busybox,/data/test.txt))
-$(eval $(call test-example-template,proxy,ros3fs,starter,/test.txt,busybox,/data/test.txt))
-$(eval $(call test-example-template,proxy,sshfs,starter,/root/sshfs-example/test.txt,busybox,/data/test.txt))
-$(eval $(call test-example-template,starter,ros3fs,starter,/test.txt,busybox,/data/test.txt))
-$(eval $(call test-example-template,starter,sshfs,starter,/root/sshfs-example/test.txt,busybox,/data/test.txt))
+$(eval $(call test-example-template,proxy,mountpoint-s3,deploy.yaml,starter,/test.txt,busybox,/data/subdir/test.txt))
+$(eval $(call test-example-template,proxy,goofys,deploy.yaml,starter,/test.txt,busybox,/data/subdir/test.txt))
+$(eval $(call test-example-template,proxy,s3fs,deploy.yaml,starter,/test.txt,busybox,/data/subdir/test.txt))
+$(eval $(call test-example-template,proxy,ros3fs,deploy.yaml,starter,/test.txt,busybox,/data/subdir/test.txt))
+$(eval $(call test-example-template,proxy,sshfs,deploy.yaml,starter,/root/sshfs-example/subdir/test.txt,busybox,/data/subdir/test.txt))
+$(eval $(call test-example-template,starter,ros3fs,deploy.yaml,starter,/test.txt,busybox,/data/subdir/test.txt))
+$(eval $(call test-example-template,starter,sshfs,deploy.yaml,starter,/root/sshfs-example/subdir/test.txt,busybox,/data/subdir/test.txt))
+ifdef TEST_SUBPATH
```

Comment:
Shall we test subpath volumes in Github Actions? 

The default kind version in the Github-Hosted runner([`ubuntu-latest`](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md#:~:text=jq%201.7-,Kind,-0.24.0)) already supports sidecar containers .

Reply:
Of course, thanks!

BTW: It would be better to recommend users to use sidecar container as a default usage.

Reply:
OK. I will create another PR for it.

Reply:
I opened https://github.com/pfnet-research/meta-fuse-csi-plugin/pull/8